### PR TITLE
Temporal Cloud limits sheet - remove KB

### PR DESCRIPTION
**Pull request by rachfop**
## What does this PR do?

Split the single temporal platform limits KB articles into two nodes.
Reformat the nodes and add headers.
Change the paragraph talking about “on this page” to “in this section”
Add a redirect with vercel.
Fix any direct links to KB article.

### CODE REVIEW

File: docs-src/cloud/default-limits-temporal-cloud.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+title: Default limits for Temporal Cloud
+sidebar_label: Default limits
+description: Temporal Cloud sets default limits for the following aspects; Account level, Namespace level, List Filters.
+tags:
+  - error
+  - warn
+  - limits
+---
+
+Temporal Cloud sets default limits for the following aspects: [Account level](#account-level), [Namespace level](#namespace-level), [List Filters](#list-filters).
```
Comment (DJSanti) [inline docs-src/cloud/default-limits-temporal-cloud.md:12]:
```suggestion
Temporal Cloud sets default limits for [Account level](#account-level), [Namespace level](#namespace-level), and [List Filters](#list-filters).
```

File: docs-src/cloud/default-limits-temporal-cloud.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+  - datetime: 20
+  - int: 20
+  - keyword: 20
+  - text: 5
+- Retention Period: 30 days (configurable; range of 1—90 days)
+- Number of Signals received per Workflow Execution: 10,000
+
+#### List Filters
+
+The following limit is applied to List Filters:
```
Comment (DJSanti) [inline docs-src/cloud/default-limits-temporal-cloud.md:41]:
```suggestion

```

File: docs-src/cloud/default-limits-temporal-cloud.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+The **ORDER BY** operator isn't supported in List Filters in Temporal Cloud.
+This means that you can't apply custom ordering of Workflows with Cloud Visibility features.
+Lists of Workflows are still ordered by a default ordering rule.
+Be aware that this rule might change.
+
+#### Increasing limits
+
+Some of the Temporal Cloud limits mentioned above can be increased by [submitting a support ticket](/cloud/introduction/support#support-ticket).
```
Comment (DJSanti) [inline docs-src/cloud/default-limits-temporal-cloud.md:50]:
```suggestion
Some of the default Temporal Cloud limits can be increased by [submitting a support ticket](/cloud/introduction/support#support-ticket).
```

File: docs-src/cloud/default-limits-temporal-cloud.md [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+
+The default values are intended to guard against unintentional resource usage.
```
Comment (DJSanti) [inline docs-src/cloud/default-limits-temporal-cloud.md:52]: Put this sentence first.

File: docs-src/cloud/temporal-platform-limits-sheet.md [hunk 5]
```diff
@@ -1,22 +1,21 @@
...
+description: Running into limits can cause unexpected failures. Knowing the limits of Temporal can prevent that.
 tags:
   - error
   - warn
   - limits
-date: 2023-04-28T00:00:00Z
 ---
 
 Running into limits can cause unexpected failures.
 Knowing the limits of Temporal can prevent that.
```
Comment (DJSanti) [inline docs-src/cloud/temporal-platform-limits-sheet.md:14]:
```suggestion
Running into limits can cause unexpected failures.
Knowing Temporal's limits can prevent that.

```

File: docs-src/concepts/what-is-an-update.md [hunk 6]
```diff
@@ -27,7 +27,7 @@ An Update has four phases.
 
 1. **Admission.** The Temporal Cluster first validates Update submissions against the configured resource usage limits.
    For example, limits apply to concurrent requests and requests per second.
-   See the [Temporal Platform limits sheet](/kb/temporal-platform-limits-sheet) for more details.
+   See the [Temporal Platform limits sheet](/cloud/introduction/operating-envelope#limits-sheet) for more details.
```
Comment (DJSanti) [inline docs-src/concepts/what-is-an-update.md:30]:
```suggestion
   See the [Temporal Platform limits sheet](/cloud/introduction/operating-envelope#limits-sheet) for more information concerning usage limits.
```

File: docs-src/references/dynamic-configuration.md [hunk 7]
```diff
@@ -193,7 +193,7 @@ The Persistence store in the Cluster has default size limits set for optimal per
 The default values on these keys are based on extensive testing.
 You can change these values, but ensure that you are provisioning enough database resources to handle the changed values.
```
Comment (DJSanti) [inline docs-src/references/dynamic-configuration.md:194]:
```suggestion
When changing these values, make sure that you're provisioning enough database resources to handle these changes.
```

**Review by DJSanti — CHANGES_REQUESTED** [review]

**Review by flossypurse — CHANGES_REQUESTED** [review]
Nice, will review soon.

**Pull request closed without merging**